draft-ietf-tls-dtls-connection-id.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: The Datagram Transport Layer Security (DTLS) Connection Identifier
-abbrev: DTLS Connection ID
+title: The Datagram Transport Layer Security (DTLS) 1.2 Connection Identifier
```

Comment:
Why 1.2? Can't this be for < 1.3?

Reply:
I pondered on this as well, then noted that 1.0 is "obsoleted" by 1.2 and decided that we probably should not care about 1.0?  Happy to revert as you suggest if you think it's the sensible thing to do.

Reply:
I think I would just remove the "1.2" here.

Reply:
ACK